Block/AdminSearchBlockService.php

```diff
@@ -91,7 +91,7 @@ public function getName()
     /**
      * {@inheritdoc}
      */
-    public function setDefaultSettings(OptionsResolverInterface $resolver)
+    public function configureSettings(OptionsResolverInterface $resolver)
```

Comment:
See method signature
https://github.com/sonata-project/SonataBlockBundle/pull/204/files#diff-0470a0f1cfc16aa469b68ab7066b5a46R37

```php
public function configureSettings(OptionsResolver $resolver)
```
so use `OptionsResolver ` instead of `OptionsResolverInterface`